assets/js/phoenix_live_view/dom_patch.js

```diff
@@ -284,7 +284,7 @@ export default class DOMPatch {
 
     if(externalFormTriggered){
       liveSocket.unload()
-      externalFormTriggered.submit()
+      Object.getPrototypeOf(externalFormTriggered).submit.call(externalFormTriggered)
```

Comment:
Nice fix! However, I would wonder why this code is this way if I found it later. Maybe a comment like this would be a good idea?

```suggestion
      // use prototype's submit in case there's a form control with name or id of "submit"
      // https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit
      Object.getPrototypeOf(externalFormTriggered).submit.call(externalFormTriggered)
```